Livecheckables/spice-protocol.rb

```diff
@@ -1,6 +1,6 @@
 class SpiceProtocol
   livecheck do
     url "https://www.spice-space.org/download/releases/"
-    regex(/spice-protocol-([\d.]+)\.tar\.bz2/)
+    regex(/spice-protocol-v?(\d+(?:\.\d+)+)\.t/)
```

Comment:
```suggestion
    regex(/href=.*?spice-protocol-v?(\d+(?:\.\d+)+)\.t/)
```

We should update this to only match within `href` attributes while we're at it.